### DISCUSSION TITLE
Dreo Chefmaker Pro support?

### DISCUSSION THREAD

**Discussion by pushnoi** [JeffSteinbok/hass-dreo#89] [Ideas]
Hi, any idea if you can also support the Chefmaker Pro? Would be awesome, but no idea if this is possible at all.

**Comment by iRonin** [comment]
Added basic support here https://github.com/JeffSteinbok/hass-dreo/pull/113

**Reply by pushnoi** [reply]
Is there some way I can test this already? And if so -- how?

**Reply by iRonin** [reply]
It's taking @JeffSteinbok a little bit of time to review and merge my code so for now the only way to test it is via a manual installation: either follow [this step](https://github.com/iRonin/hass-dreo/?tab=readme-ov-file#manually) or go to HACS, add custom repository https://github.com/iRonin/hass-dreo/ (as Integration) and install it (it "should" overwrite the original code).

**Reply by pushnoi** [reply]
> It's taking @JeffSteinbok a little bit of time to review and merge my code so for now the only way to test it is via a manual installation: either follow [this step](https://github.com/iRonin/hass-dreo/?tab=readme-ov-file#manually) or go to HACS, add custom repository https://github.com/iRonin/hass-dreo/ (as Integration) and install it (it "should" overwrite the original code).

Added your repository to test. Works like a charm! Finally I can automate actually turning the machine to standby after a cook, tnx!
